NetTopologySuite.IO.PostGis/PostGisWriter.cs

```diff
@@ -511,8 +511,8 @@ private static Ordinates CheckOrdinates(IGeometry geometry)
                 return CheckOrdinates(((ILineString)geometry).CoordinateSequence);
             if (geometry is IPolygon)
                 return CheckOrdinates((((IPolygon)geometry).ExteriorRing).CoordinateSequence);
-            if (geometry is IGeometryCollection)
-                return CheckOrdinates(geometry.GetGeometryN(0));
+            if (geometry is IGeometryCollection collection)
```

Comment:
Small nit: depending on #6 we may need to go over the entire file to do this kind of thing. In the meantime let's see what the NTS folks say.

Reply:
> go over the entire file to do this kind of thing

I'm planning to do that to the whole file once things settle down a bit, more specifically changing it all to `switch` / `case`